mesos-simulation/src/main/scala/mesosphere/mesos/simulation/SimulatedDriver.scala

```diff
@@ -118,4 +118,6 @@ class SimulatedDriver(driverProps: Props) extends SchedulerDriver with StrictLog
         Status.DRIVER_STOPPED
     }
   }
+
+  override def updateFramework(frameworkInfo: FrameworkInfo, collection: util.Collection[String]): Status = ???
```

Comment:
I'm confused by this method. What's its purpose?

Reply:
I have to implement it for sbt to build